README.md

```diff
@@ -110,12 +112,17 @@ const sendOptions = {
   ]
 };
 
-await this.nodemailer.sendMail(sendOptions);
+fastify.nodemailer.sendMail(sendOptions, (err, info) => {
+  if (err) next(err)
+  reply.send({
+    messageId: info.messageId
+  })
+})
```

Comment:
In this form we don't have `next` and `reply` here. Can you wrap this into a route declaration, please? Eg.:
```js
fastify.get('/sendmail/:email', (req, reply, next) => {
  fastify.nodemailer.sendMail(sendOptions, (err, info) => {
    if (err) next(err)
    reply.send({
      messageId: info.messageId
    })
  })
})
```

Reply:
See last commit, should to the trick...

Reply:
Will wait for your OK before completing backports...

Reply:
It would be great, i can review them tomorrow (we are in different time zone).